pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/log/RedisConnectionValidatorAppender.java

```diff
@@ -1,12 +0,0 @@
-package org.broadinstitute.ddp.log;
```

Comment:
Removing this class since it's only used in logback-based redis pings, and we are moving the pings to a separate thread outside of logback.